README.md

```diff
@@ -63,6 +60,41 @@ and the [Code Conventions](https://github.com/micropython/micropython/blob/maste
 Note that MicroPython is licenced under the MIT license, and all contributions
 should follow this license.
 
+MicroPython design values
+-------------------------
+
+For its design and implementation, MicroPython aims to follow a set of values.
+Although not a strict set of rules, these values and principles serve as a
+useful guide for new and seasoned contributors, as well as maintainers.
+
+MicroPython is at heart a combination of "Micro" and "Python": it's about
+resource constrained systems running the Python programming language.  Both of
+these concepts balance off against each other in all parts of MicroPython's
+design and implementation.
+
+The key concepts that focus the development of MicroPython are:
+- Minimalism: do lots with little.
+- Efficiency, in all respects: coding, build, execution, power consumption.
+- Consistency: the whole system feels like it was designed at once.
```

Comment:
```suggestion
- Minimalism: do lots with little.
- Efficiency: in engineering, build, execution, storage, power consumption.
- Consistency: the whole system feels like it was designed at once.
```

Having the heading on 'efficiency' be longer bugs me, and it's just not necessary to disclaim that the sublist isn't exhaustive.

I'd also 'engineering' instead of 'coding' --- it's not just about composing program code, it's also everything else that contributes to fast or slow iteration times.